lib/puppet/pops/serialization/to_data_converter.rb

```diff
@@ -41,6 +43,12 @@ def initialize(options = EMPTY_HASH)
       @symbol_as_string = options[:symbol_as_string]
       @symbol_as_string = false if @symbol_as_string.nil?
 
+      @force_symbol = options[:force_symbol]
+      @force_symbol = false if @force_symbol.nil?
+
+      @silences_warnings = options[:silences_warnings]
```

Comment:
Can we rename this to `silence_warnings` instead? To be more in tune with the other options.